File: designPattern/visitor/test1/File.java
public class File extends Entry{
	String name;
	int size;

	public File(String name, int size){
		this.name = name;
		this.size = size;
	}

	public int getSize(){
		return size;
	}

	public String getName(){
		return name;
	}

	public void accept(Visitor visitor){
		visitor.visit(this);
	}
}

File: designPattern/visitor/pythonbasic/File.py
from Entry import Entry
from Element import Element


class File(Entry, Element):
    def __init__(self, name, size):
        self.name = name
        self.size = size

    def getName(self):
        return self.name

    def getSize(self):
        return self.size

    def accept(self, visitor):
        visitor.visitFile(self)